recipes/elasticconstants/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "elasticconstants" %}
+{% set version = "0.0.2" %}
+{% set sha256 = "aa9ac9170220e6c509a2e8b9351ce86326f8302c28840fa2b8a6a5292a195121" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tdegeus/elasticConstants/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python
+    - setuptools
```

Comment:
Does this really need setuptools at runtime?

Reply:
@scopatz Thanks for you review. I am currently using `pkg_resources` (shipped with setuptools) to extract the version of my library. But I agree that this is a bit convoluted (I did not realise before packaging here that `pkg_resources` is not standard). So what I will do is update the source, and remove the dependency here, which is much cleaner.